.github/workflows/build.yml

```diff
@@ -12,11 +12,11 @@ env:
     cmake .. &&
     make &&
     make install
-  # Don't support x86, because I had problems getting a successful build.
+
   CIBW_ARCHS_LINUX: x86_64
   # A smoke test for each wheel.
   CIBW_TEST_COMMAND: >
-    python -c 'import pycodec2'
+    python -c "import sys;print(sys.path);import pycodec2"
```

Comment:
Does the print do any meaningful test that the old one didn’t catch? If not, keep the old one.